plugins/woocommerce/includes/wc-rest-functions.php

```diff
@@ -373,3 +373,54 @@ function wc_rest_check_product_reviews_permissions( $context = 'read', $object_i
 function wc_rest_is_from_product_editor() {
 	return isset( $_SERVER['HTTP_X_WC_FROM_PRODUCT_EDITOR'] ) && '1' === $_SERVER['HTTP_X_WC_FROM_PRODUCT_EDITOR'];
 }
+
+/**
+ * Check if a REST namespace should be loaded. Useful to maintain site performance even when lots of REST namespaces are registered.
+ *
```

Comment:
Maybe add a `@since` tag here?

Reply:
Added in https://github.com/woocommerce/woocommerce/pull/47704/commits/8aae39f52503c84409b43b3318151764caf542d5